jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -595,8 +595,13 @@ static List<String> computeEntrypoint(
             || !rawConfiguration.getJvmFlags().isEmpty()
             || !rawExtraClasspath.isEmpty()
             || rawConfiguration.getExpandClasspathDependencies())) {
+      // in a multi-module project it is not rare to configure a ton of jvmflags, sometimes
+      // classpath
+      // but having a submodule overriding entrypoint and ignoring all that is intended when such a
+      // config is done
+      // so just log it for user awareness but as an INFO, it is NOT an error
```

Comment:
nit: Thanks for documenting this! This comment may be better suited as a PR description. 
Comments are usually recommended when it is not entirely clear what the code is doing but in this case, the use of `LogEvent.info` is self-explanatory so it should be safe to omit it.

Reply:
Intent was to document it shouldnt be switched to warn in code - not sure anyone would use blame to check that but happy to drop it if you think it is better

Reply:
> Intent was to document it shouldnt be switched to warn in code -

I see! Increasing the log severity (to warnings or errors) needs to be done thoughtfully regardless so this shouldn't be a major concern. This change would also help us make the style be consistent with other classes in the repo.

Reply:
Done!